src/components/MerchantsPage/gam/GiftAMealPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './styles.module.scss';
+import CampaignListItem from './CampaignListItem'
+
+import gam_hero from '../images/gam_hero.png'
+import gam_1 from '../images/gam_1.svg'
```

Comment:
nit: maybe we can give these more descriptive names?

Reply:
I agree! I'll rename these.